Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionConfigurationChecker.cs

```diff
@@ -333,7 +344,8 @@ private static string GetPathDifference()
         /// the MRTK source is from the repo, warnings are converted to errors. Warnings are not converted to errors if the MRTK source is from the unity packages.
         /// Warning 618 and 649 are logged when the Leap Motion Core Assets are imported into the project, 618 is the obsolete warning and 649 is a null on start warning.
         /// </summary>
-        [MenuItem("Mixed Reality Toolkit/Utilities/Leap Motion/Configure CSC File for Leap Motion")]
+        /// <remarks>Updating the CSC file was only required for the 4.4.0 Leap Assets and only version 4.5.0 and up is supported moving forward</remarks>        
+        [Obsolete]
```

Comment:
Nit: You can add also go like:

[Obsolete("Updating the CSC file was only required for the 4.4.0 Leap Assets and only version 4.5.0 and up is supported moving forward")]

That way when you have warnings that show up it will actually say what's obsolete